www/templates/customer-balance.html

```diff
@@ -1,4 +1,10 @@
 <div>
-  <p>Balance: {{customer.balance | currency}} </p>
-  <p>Rewards: {{customer.rewards | currency}} </p>
+  <p>
+    {{ "balance" | translate }}{{ "colon" | translate }}
```

Comment:
You don't need to translate colons. Please remove all of these and the key from the json file.